src/Univapay/Resources/TransactionToken.php

```diff
@@ -159,6 +159,12 @@ public function createCharge(
             : []
         );
 
+        if ($this->paymentType === PaymentType::CARD()) {
+            $payload += isset($threeDS)
+                ? ['three_ds' => $threeDS->jsonSerialize()]
+                : ['three_ds' => ['mode' => ThreeDSMode::SKIP()->getValue(), 'redirect_endpoint' => null]];
+        }
```

Comment:
will remove this one once https://github.com/univapaycast/gyron-payments-api/pull/7761 is merged

Reply:
I think you can remove this already